airbyte-integrations/connectors/source-persistiq/integration_tests/expected_campaigns_stream.txt

```diff
@@ -0,0 +1 @@
+{"stream": "campaigns", "data": {"id": "c_N2G2Mx","name": "Streamer Outreach","creator": {"id": "u_3an2Jp","name": "Gabriel Rossmann","email": "gabriel@punctual.cc"},"stats": {"prospects_contacted": 1,"prospects_reached": 1,"prospects_opened": 1,"prospects_replied": 0,"prospects_bounced": 0,"prospects_optedout": 0,"total_prospects": 1}}, "emitted_at": 1629119628000}
```

Comment:
This expected data is specific to your PersistIq account, I added the records we have on the Airbyte sandbox account in a commit on my side branch that I'll push to yours once you give us maintainer permissions on your fork.

Reply:
Should be good 
![image](https://user-images.githubusercontent.com/22979684/150148900-37db2a64-9670-4c23-a765-210a3ae04bc4.png)